src/irvisualizer/StmtToViz_javascript.template.html

```diff
@@ -1,4 +1,3 @@
-<script>
     /* Highlighting 'matched' elements in IR code */
```

Comment:
> Just to complete: right now, you also have a nested <script> tag, because it's once in the HTML file, and once in the C++ generator code.

Tagging @mcourteaux to verify whether the nested `<script>` tags are introduced in the Halide build process.

Reply:
Sorry, was confused. Indeed, you deleted it, not added... Got the colors backwards, my bad.